memory/src/main/java/com/yahoo/memory/NativeMemory.java

```diff
@@ -129,7 +129,11 @@ public NativeMemory(final long[] longArray) {
   /**
    * Provides access to the backing store of the given writable ByteBuffer using Memory interface.
    * @param byteBuf the given writable ByteBuffer
+   *
+   * @deprecated use {@link #wrap(ByteBuffer)} which supports both readable and writable
```

Comment:
In this sentence: "which" requires a comma, ", which", and "read-only" is more precise.  Replace with
   * @deprecated Replaced by {@link #wrap(ByteBuffer)}, which handles both writable and read-only
   * ByteBuffers.